wp-content/themes/bpr2018/resources/assets/sass/_elements.scss

```diff
@@ -40,6 +39,18 @@
   }
 }
 
+@media (prefers-color-scheme: dark) {
```

Comment:
To keep things concise and grouped together, you could pull these media queries into `.pullquote` above like so:

```scss
.pullquote {
  font-family: $header-font;
  font-style: italic;
  font-weight: 700;
  font-size: 1.6em;
  margin: 0 0 1.4em 0;

  &.left {
    text-align: left;
    @media (min-width: $width-xl) { margin: 0 8vw 1.4em -8vw; }
  }
  &.right {
    text-align: right;
    @media (min-width: $width-xl) { margin: 0 -8vw 1.4em 8vw; }
  }

  @media (prefers-color-scheme: dark) {
    color: #9F9F9F;
  }
  @media (prefers-color-scheme: light) {
    color: #727272;
  }
}
```